### ISSUE TITLE
`antigen.gene` / `antigen.species` mix up for some human epitopes

### ISSUE THREAD

**Issue by JamieHeather** [antigenomics/vdjdb-db#368]
I've just updated an analysis to use the May 2024 release and noticed that there's a few entries for anti-human TCRs that appear to have the `antigen.gene` and `antigen.species` fields swapped around. There's just under 90 of them, from a handful of different sources - see [switched-antigen-gene-species.txt](https://github.com/antigenomics/vdjdb-db/files/15501583/switched-antigen-gene-species.txt).

(In double checking this I also noticed that sometimes a human protein name is used instead of the gene name in the `antigen.gene` field, e.g. p53 instead of TP53 or NY-ESO-1 instead of CTAG1B. I'm not sure if it matters, but I guess it's worth maybe flagging for people who want to cross-reference between datasets by gene symbols.)

**Comment by mikessh** [comment]
Yep, just spotted that too, sorry - got a bit distracted with Docker issues) It matters as some "self-antigens" become missing. Thanks for the fix!
